db/db_test_util.h

```diff
@@ -276,7 +276,10 @@ class SpecialEnv : public EnvWrapper {
         while (env_->delay_sstable_sync_.load(std::memory_order_acquire)) {
           env_->SleepForMicroseconds(100000);
         }
-        Status s = base_->Sync();
+        Status s;
+        if (!env_->if_skip_fsync_) {
+          s = base_->Sync();
+        }
```

Comment:
Just to confirm: do we still want to do the sleep here if the flag is set?

Reply:
I read the code and it feels that the sleep was to prevent any LSM-tree file change in some stage of some test cases. So I believe it is orthogonal to this fsync.